astro/src/styles/bootstrap.scss

```diff
@@ -313,17 +323,28 @@ $theme-colors-border-subtle-dark: map-merge($theme-colors-border-subtle-dark, $c
   color: $body-color;
 }
 
+#mc_embed_signup input.mce_inline_error {
+  border-color: $danger-border-subtle !important;
+}
+
 textarea, input {
   border-width: 1.5px 1.5px;
   border-color: #000000;
 }
 
-input[type=checkbox], input[type=radio] {
-  accent-color: #041058;
+.form-control, .check, .check:focus {
+  border-color: $info-border-subtle;
 }
 
 .check {
```

Comment:
Instead of making all radio and checkboxes implement this class, let's just put it on the element itself.
```
input[type=radio], input[type=checkbox] ...
```